docs/index.html

```diff
@@ -88,13 +89,148 @@ <h2 id="getting-started">Getting started</h2>
 </ul></li>
 </ol>
 
-<p>At this point, if you haven&rsquo;t already, you should fire up some resources that Wash can actually introspect. Otherwise, as Han Solo would say, &ldquo;this is going to be a real short trip&rdquo;. So fire up some Docker containers, create some EC2 instances, toss some files into S3, launch a Kubernetes pod, etc.</p>
+<p>At this point, if you haven&rsquo;t already, you should fire up some resources that Wash can actually introspect. Otherwise, as Han Solo would say, &ldquo;this is going to be a real short trip&rdquo;. So fire up some Docker containers, create some EC2 instances, toss some files into S3, launch a Kubernetes pod, etc. We&rsquo;ve also provided a <a href="#wash-by-example">guided tour</a> that includes a Docker application.</p>
```

Comment:
Change the first use of "fire up" in this paragraph to just plain old "start," otherwise the idiom loses its punch. Keep "So fire up some Docker containers..."